flang/lib/Lower/Bridge.cpp

```diff
@@ -5078,6 +5088,13 @@ class FirConverter : public Fortran::lower::AbstractConverter {
   /// intended for device offloading has been detected
   bool ompDeviceCodeFound = false;
 
+  /// Keeps track of symbols defined as declare target that could not be
+  /// processed at the time of lowering the declare target construct, such
+  /// as certain cases where interfaces are declared but not defined within
+  /// a module.
+  llvm::SmallVector<Fortran::lower::OMPDeferredDeclareTargetInfo, 2>
```

Comment:
I think we should be able to use the default now.
```suggestion
  llvm::SmallVector<Fortran::lower::OMPDeferredDeclareTargetInfo>
```